### DISCUSSION TITLE
Missing video?

### DISCUSSION THREAD

**Discussion by aronvandepol** [mrdbourke/pytorch-deep-learning#66] [Q&A] [answered]
In Section 02: PyTorch Neural Network Classification we are coding our model using Class CircleModel etc...

Up until the video 'Making our Neural Network Visual' i have the same code and could follow along. Yet suddenly, in 'Setting Up a Loss Function Optimizer and Evaluation Function for Our Classification Network' we see the model has been build differently using `nn.Sequential`. with the comment 'lets replicate our model using nn.Sequential'.

I have gone over the videos prior a dozen times now and cant figure out where i missed this. Is perhaps this part in the video courses missing?

Any help is appreciated! loving the course so far.

**Comment by mrdbourke** [comment]
I just checked back over the videos, and you're right, it seems there's a missing video 🤔 

Let me go back through the archives and see if I can find it, because the code is definitely still in the notebook: https://github.com/mrdbourke/pytorch-deep-learning/blob/main/video_notebooks/02_pytorch_classification_video.ipynb 

If I can't find it, I'll rerecord it ASAP.

Thank you for letting me know!

PS I'll post back here once it's recorded/up.

**Reply by mrdbourke** [reply]
**Update:** Found the missing video!

Turns out when I recorded/saved it I saved it with the wrong numbering/in a different folder to all the other videos (no idea why).

I'll get it edited and uploaded ASAP.

<img width="536" alt="Screen Shot 2022-07-12 at 10 24 46 am" src="https://user-images.githubusercontent.com/16750345/178380901-8a62411f-ca3a-420e-bee5-39d8cb9167d8.png">

**Reply by aronvandepol** [reply]
Thank you! That must be it then :)

**Comment by mrdbourke** [comment, marked as answer]
Hey @aronvandepol, the video you mentioned should now be uploaded and available :) 

Thank you again for letting me know.

And if you find anything else missing, don't hesitate to reach out.

<img width="1500" alt="Screen Shot 2022-07-13 at 4 00 05 pm" src="https://user-images.githubusercontent.com/16750345/178661669-81759313-db1b-45ef-b872-455b2b673097.png">
